### PR TITLE
Constructors, Destructors and Assignment Operators

### PR DESCRIPTION
**Pull request by elixir14082002**
<hr>

## Description 📜

Included documentation and video on Constructors, Destructors, and Assignment Operators.
#3035

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [x] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by elixir14082002** [comment]
@Suvraneel @kashishahuja2002 I have made the requested changes in this PR

**Review by gunjan-g — CHANGES_REQUESTED** [review]
@elixir14082002 Please move your file Constructor_Destructor in the Constructor-Destructor folder only instead of C++ 14
and add index line along with relative link in readme properly, you haven't added that

**Comment by gunjan-g** [comment]
@elixir14082002 this topic is already there in repo, you can just make a pr of the video part if you want and that too on a separate issue, if it's not been taken
Ps. Please check the issues and branch C/CPP before working on any topic

**Comment by elixir14082002** [comment]
@gunjan-g The topic is a little different and I asked the supervisors before starting my work. I have removed it from C++14.

**Comment by gunjan-g** [comment]
> @gunjan-g The topic is a little different and I asked the supervisors before starting my work. I have removed it from C++14.

Constructors and destructors are there, we can't repeat the topic. Let me know how this is different from the content present. You can add assignment operators but you have assigned an issue opened by someone else, so you have to open a different issue for that too

**Comment by elixir14082002** [comment]
> > @gunjan-g The topic is a little different and I asked the supervisors before starting my work. I have removed it from C++14.
> 
> Constructors and destructors are there, we can't repeat the topic. Let me know how this is different from the content present. You can add assignment operators but you have assigned an issue opened by someone else, so you have to open a different issue for that too

Actually, I was told that I can work on this particular issue, I only need to change the information a little, and also there is the topic of the assignment operators. That's why I created the PR. This was an open issue and no one was working on it. @gunjan-g

File: C_CPP/Constructor & Destructor/Constructor_Destructor.md
Comment (kashishahuja2002) [inline C_CPP/Constructor & Destructor/Constructor_Destructor.md:11]: Remove this line.

File: C_CPP/Constructor & Destructor/readme.md
Comment (kashishahuja2002) [inline C_CPP/Constructor & Destructor/readme.md:1]: Remove this file.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Why are there 3 files?
Combine the content in one.

**Comment by anisha-01-phoenix** [comment]
Looks Good!...Can be Merged!

**Comment by kashishahuja2002** [comment]
This topic already exists in doc mode
so you either need to change it to video mode or work on a  new issue.

**Comment by kashishahuja2002** [comment]
> Looks Good!...Can be Merged!

Ma'am, You're a contributor, not a mentor/supervisor. You're approval is neither required nor appreciated here. I would suggest you to leave it upto mentors and supervisors, and kindly focus on contributing. :)

**Comment by gunjan-g** [comment]
> Looks Good!...Can be Merged!

So sweet of you, handling the work of a mentor and contributor at the same time. Honestly, we were waiting for your approval only XD

**Comment by elixir14082002** [comment]
I am closing this PR due to several issues.

**Pull request closed without merging**
